src/UIContainer.ts

```diff
@@ -888,8 +891,13 @@ export class UIContainer extends HTMLElement {
         if (this.userIdleTimeout < 0) {
             return;
         }
-
         this.setAttribute(Attribute.USER_IDLE, '');
+
+        // Blur active element so that first key press on TV doesn't result in an action.
+        const focusedChild = getFocusedChild(getFocusableChildren(this));
+        if (this.deviceType === 'tv' && focusedChild !== null && this.isUserIdle_()) {
```

Comment:
Move the `this.deviceType === 'tv' && this.isUserIdle_()` check to before the `const focusedChild = ...` line, so we don't unnecessarily compute the focused child if we're not going to use it.